src/test/java/seedu/address/logic/commands/AdjustCommandTest.java

```diff
@@ -0,0 +1,95 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.Transcript;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.module.Code;
+import seedu.address.model.module.Grade;
+import seedu.address.model.module.Module;
+import seedu.address.model.util.ModuleBuilder;
+
+public class AdjustCommandTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private CommandHistory commandHistory = new CommandHistory();
+
+    private Module moduleAbc = new ModuleBuilder().withCode("ABC").withYear(2).noGrade().build();
```

Comment:
I feel that it is better to have a central class in this case that keeps the definition of the modules so that anyone changing any fields of module would not have to go through and your code so that all test passes.